towhee/functional/mixins/dataset.py

```diff
@@ -25,6 +26,7 @@ class DatasetMixin:
 
     # pylint: disable=import-outside-toplevel
     @classmethod
+    @register_dag
```

Comment:
we don't need to register a data source as DAG

Reply:
fixed

Reply:
fixed